events.blade.php

```diff
@@ -84,6 +84,26 @@ protected function getListeners()
 @endslot
 @endcomponent
 
+### Using listeners in a Trait
+
+You can also add listeners in a Trait. All you need to do is to merge them with the eventual existing listeners of the component in your Trait's mount method:
+
+@component('components.code-component')
+@slot('class')
+class MyTrait {
```

Comment:
Change this to `trait MyTrait`

Reply:
Well there I go with copying and pasting from the docs :)

But it's fixed!